README.md

```diff
@@ -118,6 +118,10 @@ I renamed the repository to "Awesome Alternatives in Rust". The original name wa
 * [bottom](https://github.com/ClementTsang/bottom) - Yet another cross-platform graphical process/system monitor.
 * [ytop](https://github.com/cjbassi/ytop) (no longer maintained) - A TUI system monitor written in Rust
 
+#### [httpie](https://github.com/httpie/httpie)
+
+* [xh](https://github.com/ducaale/xh) - Friendly and fast tool for sending HTTP requests
```

Comment:
Could you follow the alphabet order? In this case, you should add httpie below hexdump.

Reply:
Done. fixed in commit 920c9b8252552d3503fa26e96c2896298b3e256c

Reply:
Thank you!